README.md

```diff
@@ -89,6 +89,37 @@ curl -k -X POST https://localhost:8090/oauth2/token \
   - **Client ID:** `client123`
   - **Client Secret:** `secret123`
 
+#### 3️⃣ Configure Login with Github
+
+- Create an OAuth application in your Github account following the instructions given in the [Github documentation](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/creating-an-oauth-app).
+  - Configure the urls as follows:
+    - **Homepage URL:** `https://localhost:8090`
+    - **Authorization callback URL:** `https://localhost:8090/flow/authn`
+  - Copy the **Client ID** and **Client Secret**.
+
+- Open the deployment.yaml file in the `backend/cmd/server/repository/conf` directory and add the following configurations:
+
+  ```yaml
+  authenticator:
+    default: "GithubAuthenticator"
+    authenticators:
+      - name: "GithubAuthenticator"
+        id: "147b3c7e-0a2f-4d8b-9c5f-a03800000002"
```

Comment:
Why we need to get this id from the config?

Reply:
Modeled it to the authenticator id. However this can be removed.
Removed with 96263715f1c111ffc5f201819b1c43a1a6eeeb21